core/rawdb/accessors_chain_op_test.go

```diff
@@ -0,0 +1,74 @@
+		"0xf901a2f9019f018303183df90197f89b9425e1c58040f27ecf20bbd4ca83a09290326896b3f863a00109fc6f55cf40689f02fbaad7af7fe7bbac8a3d2186600afc7d3e10cac60271a0000000000000000000000000000000000000000000000000000000000000c026a0000000000000000000000000689d0367b9d654aae886982894896f3a826840eda000000000000000000000000000000000000000000000000000000000618d8837f89c9425e1c58040f27ecf20bbd4ca83a09290326896b3f884a092e98423f8adac6e64d0608e519fd1cefb861498385c6dee70d58fc926ddc68ca00000000000000000000000000000000000000000000000000000006e77d5ea1ca0000000000000000000000000000000000000000000000000000000000000c026a0000000000000000000000000689d0367b9d654aae886982894896f3a826840ed80f85a9425e1c58040f27ecf20bbd4ca83a09290326896b3f842a0fe25c73e3b9089fac37d55c4c7efcba6f04af04cebd2fc4d6d7dbb07e1e5234fa000000000000000000000000000000000000000000000005b3c134de04354800080",
+		"0xf90104f90101018301c60df8faf89c9425e1c58040f27ecf20bbd4ca83a09290326896b3f884a092e98423f8adac6e64d0608e519fd1cefb861498385c6dee70d58fc926ddc68ca00000000000000000000000000000000000000000000000000000006ec9dbc284a0000000000000000000000000000000000000000000000000000000000000c026a000000000000000000000000070b17c0fe982ab4a7ac17a4c25485643151a1f2d80f85a9425e1c58040f27ecf20bbd4ca83a09290326896b3f842a0fe25c73e3b9089fac37d55c4c7efcba6f04af04cebd2fc4d6d7dbb07e1e5234fa000000000000000000000000000000000000000000000005b3a1f1505a2f4800080",
+		"0xf90104f90101018301c60df8faf89c9425e1c58040f27ecf20bbd4ca83a09290326896b3f884a092e98423f8adac6e64d0608e519fd1cefb861498385c6dee70d58fc926ddc68ca00000000000000000000000000000000000000000000000000000006e89bf95dea0000000000000000000000000000000000000000000000000000000000000c026a0000000000000000000000000e5e7492282fd1e3bfac337a0beccd29b15b7b24080f85a9425e1c58040f27ecf20bbd4ca83a09290326896b3f842a0fe25c73e3b9089fac37d55c4c7efcba6f04af04cebd2fc4d6d7dbb07e1e5234fa000000000000000000000000000000000000000000000005b382adc2b0294800080",
+		"0xf90104f90101018301c60df8faf89c9425e1c58040f27ecf20bbd4ca83a09290326896b3f884a092e98423f8adac6e64d0608e519fd1cefb861498385c6dee70d58fc926ddc68ca00000000000000000000000000000000000000000000000000000006ea50c77c9a0000000000000000000000000000000000000000000000000000000000000c026a00000000000000000000000002539ffc9ded82926a5aaee065e800c7d1de0245480f85a9425e1c58040f27ecf20bbd4ca83a09290326896b3f842a0fe25c73e3b9089fac37d55c4c7efcba6f04af04cebd2fc4d6d7dbb07e1e5234fa000000000000000000000000000000000000000000000005b3636a3506234800080",
+	}
+
+	for i, rawDBReceiptHex := range rawDBReceiptHexes {
+		t.Run(fmt.Sprintf("OPM block %d", i), func(t *testing.T) {
+			rawDBReceiptBytes, err := hexutil.Decode(rawDBReceiptHex)
+			require.NoError(t, err)
+
+			sr := new([]receiptLogs)
```

Comment:
I think this should be

```suggestion
			sr := make([]*types.ReceiptForStorage, 0)
```

Reply:
Implemented here https://github.com/ethereum-optimism/op-geth/pull/738

Reply:
I think it would trigger the same RLP decoder. I think I chose to do it this way since https://github.com/ethereum-optimism/op-geth/pull/726 just changes the implementation of the decoder on `receiptLogs` to use `ReceiptForStorage`.

https://github.com/ethereum-optimism/op-geth/blob/472228745b7e5c6d777897c83067fe9dacc47ad3/core/rawdb/accessors_chain.go#L673-L681